alignai/validate.py

```diff
@@ -0,0 +1,32 @@
+class ValidationError(Exception):
```

Comment:
```suggestion
class ValidationError(ValueError):
```

> Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as [IndexError](https://docs.python.org/3.12/library/exceptions.html#IndexError).

nit. 좀 더 Clear 하게 `ValueError` 를 상속하는 건 어떨까요? 의미적으로 가까운 built-in Exception 인 것 같아서요.

Reply:
좋습니다!!